share/spack/gitlab/cloud_pipelines/stacks/e4s-power/spack.yaml

```diff
@@ -0,0 +1,424 @@
+  - ginkgo +cuda cuda_arch=70
+  - heffte +cuda cuda_arch=70
+  - hpctoolkit +cuda
+  - hpx +cuda cuda_arch=70
+  - hypre +cuda cuda_arch=70
+  - kokkos +wrapper +cuda cuda_arch=70
+  - kokkos-kernels +cuda cuda_arch=70 ^kokkos +wrapper +cuda cuda_arch=70
+  - magma +cuda cuda_arch=70
+  - mfem +cuda cuda_arch=70
+  - papi +cuda
+  - parsec +cuda cuda_arch=70
+  - petsc +cuda cuda_arch=70
+  - raja +cuda cuda_arch=70
+  - slate +cuda cuda_arch=70
+  - slepc +cuda cuda_arch=70
+  - strumpack ~slate +cuda cuda_arch=70
+  - sundials +cuda cuda_arch=70
+  - superlu-dist +cuda cuda_arch=70
+  - tasmanian +cuda cuda_arch=70
+  - tau +mpi +cuda
+  - trilinos@13.4.0 +cuda cuda_arch=70
+  - umpire ~shared +cuda cuda_arch=70
+  - vtk-m +cuda cuda_arch=70
+  - zfp +cuda cuda_arch=70
+
+  # CPU failures
+  # - bricks           # bricks: VSBrick-mpi9pt.py-Scalar-4x4x4x4-1:2306:17: error: 'buf0' was not declared in this scope
+  # - geopm            # /usr/include/ppc64le-linux-gnu/bits/string_fortified.h:95:10: error:'__builtin_strncpy' specified bound 512 equals destination size [-Werror=stringop-truncation]
+  # - loki             # ../include/loki/Singleton.h:158:14: warning: 'template<class> class std::auto_ptr' is deprecated: use 'std::unique_ptr' instead [-Wdeprecated-declarations]
+  # - paraview +qt     # llvm@14
+  # - variorum         # /usr/bin/ld: Intel/CMakeFiles/variorum_intel.dir/Broadwell_4F.c.o:(.bss+0x0): multiple definition of `g_platform';
+  # ----
+
+  # CUDA failures
+  # - dealii +cuda cuda_arch=70           # fltk
+  # - bricks +cuda                        # bricks
+  # ----
+  # fltk: gl_draw.cxx:120:13: warning: 'void get_list(int)' defined but not used [-Wunused-function]
+
+
+  mirrors: { "mirror": "s3://spack-binaries/develop/e4s-power" }
+
+  gitlab-ci:
+
+    script:
+      - . "./share/spack/setup-env.sh"
+      - spack --version
+      - cd ${SPACK_CONCRETE_ENV_DIR}
+      - spack env activate --without-view .
+      - spack config add "config:install_tree:projections:${SPACK_JOB_SPEC_PKG_NAME}:'morepadding/{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'"
+      - mkdir -p ${SPACK_ARTIFACTS_ROOT}/user_data
+      - if [[ -r /mnt/key/intermediate_ci_signing_key.gpg ]]; then spack gpg trust /mnt/key/intermediate_ci_signing_key.gpg; fi
+      - if [[ -r /mnt/key/spack_public_key.gpg ]]; then spack gpg trust /mnt/key/spack_public_key.gpg; fi
+      - spack -d ci rebuild > >(tee ${SPACK_ARTIFACTS_ROOT}/user_data/pipeline_out.txt) 2> >(tee ${SPACK_ARTIFACTS_ROOT}/user_data/pipeline_err.txt >&2)
+
+    image: ecpe4s/ubuntu20.04-runner-ppc64le:2022-08-01
+    
+    mappings:
+      - match:
+          - hipblas
+          - llvm
+          - llvm-amdgpu
+          - rocblas
+        runner-attributes:
+          tags: [ "spack", "huge", "ppc64le" ]
+          variables:
+            CI_JOB_SIZE: huge
+            KUBERNETES_CPU_REQUEST: 11000m
+            KUBERNETES_MEMORY_REQUEST: 42G
```

Comment:
I'm of two minds on including these variables in the mappings.  They are used by kubernetes in scheduling job onto pods running on aws runners.  Since we don't have any power runners in the aws cluster (and might not ever), these variables are unused in this stack and might just confuse people.  However, I could see keeping them here (and making them somewhat accurate) for consistency with other stacks and/or in case we ever got power runners in the cloud.  

I'm not requesting a change here, just thinking out loud.

Reply:
Absolutely! Good catch. I didn't mean to include these -- I was just copying the other environment and then modifying. I will remove them since they are not needed. Totally in favor of keeping things as simple as possible. Thank you.